src/Exceptions/JsonException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Metamorphosis\Exceptions;
+
+use Exception;
```

Comment:
perhaps `InvalidArgumentException`? (or smth like that)

Reply:
I'm using this class because in future versions of php it will have one with the same name, and then we can be remove this one